reference/orders.v2.oas2.yml

```diff
@@ -4295,6 +4305,17 @@ components:
                 type: integer
               value:
                 type: string
+                description: |-
+                  Depending on the option type, value can be one of the following:
+                  - The variant option value id or the modifier value id for modifier types with a list of defined values, such as drop down or checkbox modifiers.
+                  - The modifier value for modifier types without a list of defined values, such as text field or date field modifiers.
+
+                  Notes:
+                  - The following option types are not currently supported via api:
+                    - File Upload
+                    - Pick List
+                  - For date modifiers use either the `ISO-8601` or the `YYYY-mm-dd` format. The date values are saved and retuned as timestamps. For values entered using the YYYY-mm-dd format the store timezone is used when creating the timestamp.
```

Comment:
maybe we should be specific which format is supported as ISO-8601 allows quite a number of formats: 

![image](https://user-images.githubusercontent.com/63274600/128649257-1dd2a6d6-e4b2-4706-9a81-05438687ee06.png)

Reply:
TIL, thanks 👍 
I thought ISO-8601 is the complete one with TZ.
Will update it.

Reply:
How is the updated version?

Reply:
looks good now 👍 , thanks Dorna

Reply:
🙇‍♀️